src/auto-evo/simulation/food_source/HeterotrophicFoodSource.cs

```diff
@@ -38,15 +38,24 @@ public override float FitnessScore(Species species, SimulationCache simulationCa
             var predatorSpeed = microbeSpecies.BaseSpeed;
             predatorSpeed += simulationCache.GetEnergyBalanceForSpecies(microbeSpecies, patch).FinalBalance;
 
-            // It's great if you can engulf this prey, but only if you can catch it
+            // Only assign engulf score if one can actually engulf
             var engulfScore = 0.0f;
             if (microbeSpeciesHexSize / preyHexSize >
                 Constants.ENGULF_SIZE_RATIO_REQ && !microbeSpecies.MembraneType.CellWall)
             {
                 engulfScore = Constants.AUTO_EVO_ENGULF_PREDATION_SCORE;
-            }
 
-            engulfScore *= predatorSpeed > preySpeed ? 1.0f : Constants.AUTO_EVO_ENGULF_LUCKY_CATCH_PROBABILITY;
+                // The faster the better
+                engulfScore *= predatorSpeed / preySpeed;
+
+                // Allow for some degree of lucky engulfment
+                engulfScore *= predatorSpeed > preySpeed ? 1.0f : Constants.AUTO_EVO_ENGULF_LUCKY_CATCH_PROBABILITY;
+
+                // Favor larger microbes, as if they stand more chances in a fight/can digest prey faster.
+                // Here it is artificially after speed comparison for gamey incentive.
+                // Compensating lower speed requires at least PREDATION_SCORE/LUCKY_CATCH_PROBABILITY more hexes.
+                engulfScore += microbeSpeciesHexSize;
```

Comment:
I would lean towards multiplication here instead of addition, since I don't really have a sense of scale for how many points of speed is equal to one more hex size. This also means that a species with zero speed could be very effective at chasing down prey simply by being huge.

Reply:
Addition is basically intended to keep the same bonus as before, but restrict it to engulfers. On the other hand, you are right on the pitfall. I'll try that, thanks.